src/main/java/com/sap/cloudfoundry/client/facade/adapters/LogCacheClient.java

```diff
@@ -0,0 +1,42 @@
+package com.sap.cloudfoundry.client.facade.adapters;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.cloudfoundry.reactor.ConnectionContext;
+import org.cloudfoundry.reactor.HttpClientResponseWithBody;
+import org.cloudfoundry.reactor.TokenProvider;
+import org.cloudfoundry.reactor.util.AbstractReactorOperations;
+import org.cloudfoundry.reactor.util.JsonCodec;
+import org.springframework.web.util.UriComponentsBuilder;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.util.Map;
+import java.util.UUID;
+
+public class LogCacheClient extends AbstractReactorOperations {
+
+    public LogCacheClient(ConnectionContext connectionContext, String cfApi, TokenProvider tokenProvider,
+                          Map<String, String> requestTags) {
+        super(connectionContext, Mono.just(cfApi.replace("api", "log-cache")), tokenProvider, requestTags);
+    }
+
+    public Flux<ApplicationLogEntity> getRecentLogs(UUID applicationGuid) {
+        return createOperator().flatMapMany(operator -> operator.get()
+                                                                .uri(builder -> getEndpoint(builder, applicationGuid.toString()))
+                                                                .response()
+                                                                .parseBodyToMono(this::parseBody)
+                                                                .flatMapIterable(ApplicationLogsResponse::getLogs));
+    }
+
+    private UriComponentsBuilder getEndpoint(UriComponentsBuilder builder, String applicationGuid) {
+        return builder.pathSegment("api", "v1", "read", applicationGuid)
+                      .queryParam("envelope_types", "LOG");
+    }
+
+    private Mono<ApplicationLogsResponse> parseBody(HttpClientResponseWithBody response) {
+        var objectMapper = new ObjectMapper().configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
```

Comment:
Can we reuse JsonUtil instead of creating a new ObjectMapper

Reply:
The object mapper in `JsonUtil` isn't configurable.